File: s3/cycle1/ex05.c
#include <stdio.h>

int main(void)
{
	int small , secSmall , i , n, temp;
	printf("Enter the number of terms: \n");
	scanf("%d" , &n);
	printf("Enter the numbers : \n");
	scanf(" %d" , &small);
	scanf(" %d" , &temp);
	if(temp < small)
	{
		secSmall = small;
		small = temp;
	}
	else secSmall = temp;
	for(i = 0 ; i< n-2 ; i++)
	{
		scanf(" %d" , &temp);
		if(temp < secSmall && temp > small)
			secSmall = temp;
		else if(temp < small)
		{
			secSmall = small;
			small = temp;
		}
	}
	printf("\nSecond small term : %d\n", secSmall);

}

File: s3/cycle1/java/ex05.java
import java.util.*;

public class ex05{
	public static void main(String[] args)
	{
		int small , secSmall , i , n, temp;
		System.out.print("Enter the number of terms: ");
		Scanner input = new Scanner(System.in);
		n = input.nextInt();
		System.out.print("Enter the numbers : \n");
		small = input.nextInt();
		temp = input.nextInt();
		if(temp < small)
		{
			secSmall = small;
			small = temp;
		}
		else secSmall = temp;
		for(i = 0 ; i< n-2 ; i++)
		{
			int temp1 = input.nextInt();
			if(temp1 < secSmall && temp1 > small)
				secSmall = temp1;
			else if(temp1 < small)
			{
				secSmall = small;
				small = temp1;
			}
			else if(temp1 > small && small==secSmall)
				secSmall = temp1;
		}
		System.out.println("\nSecond small term : "+ secSmall);
	}
}